app/views/buildings/main/show.html.slim

```diff
@@ -129,6 +129,20 @@
         - narratives.each do |narrative|
           = render 'narratives/narrative', narrative:
 
+    - documents = @building.documents.accessible_by(current_ability)
+    .card class="#{@building.description.blank? ? 'mb-3' : 'empty-card'}"
+      .card-header
+        = link_to 'Add Documents', new_building_document_path(@building.object), class: 'float-right'
+        | Documents
+      - if documents.blank?
+        .card-body
+          p.lead No Documents yet for this building.
+
+    - if documents.present?
+      .list-group
+        - documents.each do |document|
+          = render 'documents/document', document:
```

Comment:
Indent lines 141-144 by one stop so that line 141 is in line with line 137. Otherwise, this will cause the documents list to appear outside of the card.